.github/workflows/promote_charm.yaml

```diff
@@ -10,16 +10,16 @@ on:
         type: string
         description: 'Charmcraft Base Architecture'
         default: 'amd64'
+      charm-working-directory:
+        type: string
+        description: The working directory for the charm
       destination-channel:
         type: string
         description: 'Destination Channel'
       doc-automation-disabled:
         type: boolean
         description: 'Whether to disable the documentation automation'
         default: true
-      docs-working-directory:
```

Comment:
This actually prevents the docs to be in a place different from the charm or am I missing something?